### PR TITLE
Add ability to swap distro

### PR DESCRIPTION
**Pull request by seripap**
Adds input for swappable disto.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -22,6 +22,10 @@ inputs:
     description: 'Fixed hostname to use.'
     required: false
     default: ''
+  distro:
```
Comment (DentonGentry) [inline action.yml:25]: I'd recommend naming this `arch` as `distro` more often refers to Debian vs Fedora vs Mandrake etc.

**Review by DentonGentry — CHANGES_REQUESTED** [review]

**Comment by DentonGentry** [comment]
Please also add a signed-off-by line, which can be done using:
- `git commit --amend --signoff`
- `git push -f`

File: action.yml (hunk 1 above)
Comment (seripap) [reply action.yml:25]: Sure thing, I was going by the naming here: https://pkgs.tailscale.com/stable/#static, happy to swap it.

**Comment by seripap** [comment]
Thanks for your feedback, changes added.

**Review by DentonGentry — APPROVED** [review]

**Comment by DentonGentry** [comment]
Implemented slightly differently, in https://github.com/tailscale/github-action/pull/72

**Pull request closed without merging**